src/test/java/hw10/AbstractApiTest.java

```diff
@@ -0,0 +1,52 @@
+package hw10;
+
+import static org.example.utils.Config.getApiKeyFromProperties;
+import static org.example.utils.Config.getApiTokenFromProperties;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import java.io.IOException;
+import org.example.entities.BoardEntity;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class AbstractApiTest {
+    public static BoardEntity board;
+    static final String apiKey = getApiKeyFromProperties();;
+    static final String apiToken = getApiTokenFromProperties();;
+    static final String baseUri = "https://api.trello.com/1";
+    public static String endpointBoard = "/boards";
+    public static String endpointBoardWithID = "/boards/{id}";
+    public static String expectedBoardName;
+
+    static RequestSpecification reqSpec;
+    static ResponseSpecification respSpec;
+    static ResponseSpecification negRespSpec;
+
+
+
+    @BeforeClass
+    public static void setup() {
+        reqSpec = new RequestSpecBuilder().addQueryParam("key", apiKey)
+                                          .addQueryParam("token", apiToken)
+                                          .setContentType(ContentType.JSON)
+                                          .setBaseUri(baseUri)
+                                          .build();
+        respSpec = new ResponseSpecBuilder().expectStatusCode(200)
+                                            .expectContentType(ContentType.JSON)
+                                            .build();
+        negRespSpec = new ResponseSpecBuilder().expectStatusCode(404)
+                                               //.expectContentType(ContentType.JSON)
+                                               .build();
```

Comment:
for remove code duplication you can add logging to request/response spec.
```suggestion
                                              reqSpec = new RequestSpecBuilder().addQueryParam("key", getApiKeyFromProperties())
                                          .addQueryParam("token", getApiTokenFromProperties())
                                          .setContentType(ContentType.JSON)
                                          .setBaseUri(baseUri)
                                          .build().log().ifValidationFails();

        respSpec = new ResponseSpecBuilder().expectStatusCode(200)
                                            .expectContentType(ContentType.JSON)
                                            .build().log().ifValidationFails();

        negRespSpec = new ResponseSpecBuilder().expectStatusCode(404)
                                               .build().log().ifValidationFails();
```